mmdeploy/backend/pplnn/utils.py

```diff
@@ -37,14 +37,14 @@ def register_engines(device_id: int,
         x86_engine = pplnn.X86EngineFactory.Create(x86_options)
         if not x86_engine:
             logger.error('Failed to create x86 engine')
-            sys.exit(-1)
+            sys.exit(1)
 
         if disable_avx512:
             status = x86_engine.Configure(pplnn.X86_CONF_DISABLE_AVX512)
             if status != pplcommon.RC_SUCCESS:
                 logger.error('x86 engine Configure() failed: ' +
                              pplcommon.GetRetCodeStr(status))
-                sys.exit(-1)
+                sys.exit(1)
```

Comment:
seems `1` is valid but `-1` is from [pplnn](https://github.com/openppl-public/ppl.nn/blob/01314256e227625757a7ad7aa95fb9c28dc69dc5/tools/pplnn.py#L200)